test/start_server_test.js

```diff
@@ -74,6 +74,32 @@ suite('startServer', () => {
     })
   });
 
+  suite('proxy', () =>  {
+    test('rewrites directory with proxy', (done) => {
+      startServer({
+        root: __dirname,
+        port: 8080
```

Comment:
hard coding the port, especially a common one like this, could cause the tests to fail.